<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PET_brg_ch_calibration_page</class>
 <widget class="QWidget" name="PET_brg_ch_calibration_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>1008</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Календарное время выполнения последней калибровки</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Напряжение смещения ФЭУ, наклон установки (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Опорное напряжение АЦП, наклон установки (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Флаги</string>
       </property>
      </widget>
     </item>
     <item row="27" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Напряжение смещения ФЭУ, наклон измерения (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Опорное напряжение АЦП, смещение установки (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Опорное напряжение АЦП, наклон измерения (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Смещение сигнала ФЭУ, наклон установки (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="29" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Температура платы считывания, наклон измерения (х0,01 1/°C)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Номер канала</string>
       </property>
      </widget>
     </item>
     <item row="30" column="0">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>Температура модуля сопряжения, смещение измерения (х0,01 °C)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Смещение сигнала X+ (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Смещение сигнала X- (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Смещение сигнала ФЭУ, наклон измерения (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="26" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Напряжение смещения ФЭУ, смещение измерения (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Наклон сигнала X+ (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="28" column="0">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Температура платы считывания, смещение измерения (х0,01 °C)</string>
       </property>
      </widget>
     </item>
     <item row="31" column="0">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>Температура модуля сопряжения, наклон измерения (х0,01 1/°C)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Наклон сигнала X- (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Смещение сигнала Y+ (мВ</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Наклон сигнала Y+ (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Смещение сигнала Y- (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Наклон сигнала Y- (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Поправка метки времени (пс)</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Смещение сигнала ФЭУ, смещение установки (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Смещение сигнала ФЭУ, смещение измерения (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Порог дискриминации, смещение установки (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Порог дискриминации, наклон установки (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Порог дискриминации, смещение измерения (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Опорное напряжение АЦП, смещение измерения (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Напряжение смещения ФЭУ, смещение установки (мВ)</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Порог дискриминации, наклон измерения (1/мВ)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sb_channel"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sb_flags"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sb_timestamp"/>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sb_xp_offset"/>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="sb_xp_gain"/>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="sb_xn_offset"/>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="sb_xn_gain"/>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="sb_yp_offset"/>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="sb_yp_gain"/>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="sb_yn_offset"/>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="sb_yn_gain"/>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="sb_ts_offset"/>
     </item>
     <item row="12" column="1">
      <widget class="QSpinBox" name="sb_in_offset_set_offset"/>
     </item>
     <item row="13" column="1">
      <widget class="QSpinBox" name="sb_in_offset_set_gain"/>
     </item>
     <item row="14" column="1">
      <widget class="QSpinBox" name="sb_in_offset_get_offset"/>
     </item>
     <item row="15" column="1">
      <widget class="QSpinBox" name="sb_in_offset_get_gain"/>
     </item>
     <item row="16" column="1">
      <widget class="QSpinBox" name="sb_in_threshold_set_offset"/>
     </item>
     <item row="17" column="1">
      <widget class="QSpinBox" name="sb_in_threshold_set_gain"/>
     </item>
     <item row="18" column="1">
      <widget class="QSpinBox" name="sb_in_threshold_get_offset"/>
     </item>
     <item row="19" column="1">
      <widget class="QSpinBox" name="sb_in_thresholdga_get_gain"/>
     </item>
     <item row="20" column="1">
      <widget class="QSpinBox" name="sb_adc_ref_set_offset"/>
     </item>
     <item row="21" column="1">
      <widget class="QSpinBox" name="sb_adc_ref_set_gain"/>
     </item>
     <item row="22" column="1">
      <widget class="QSpinBox" name="sb_adc_ref_get_offset"/>
     </item>
     <item row="23" column="1">
      <widget class="QSpinBox" name="sb_adc_ref_get_gain"/>
     </item>
     <item row="24" column="1">
      <widget class="QSpinBox" name="sb_pm_bias_set_offset"/>
     </item>
     <item row="25" column="1">
      <widget class="QSpinBox" name="sb_pm_bias_set_gain"/>
     </item>
     <item row="26" column="1">
      <widget class="QSpinBox" name="sb_pm_bias_get_offset"/>
     </item>
     <item row="27" column="1">
      <widget class="QSpinBox" name="sb_pm_bias_get_gain"/>
     </item>
     <item row="28" column="1">
      <widget class="QSpinBox" name="sb_readout_temp_get_offset"/>
     </item>
     <item row="29" column="1">
      <widget class="QSpinBox" name="sb_readout_temp_get_gain"/>
     </item>
     <item row="30" column="1">
      <widget class="QSpinBox" name="sb_ifmodule_temp_get_offset"/>
     </item>
     <item row="31" column="1">
      <widget class="QSpinBox" name="sb_ifmodule_temp_get_gain"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
